yt/visualization/tests/test_particle_plot.py

```diff
@@ -209,6 +217,52 @@ def test_particle_phase_plot(self):
             for fname in TEST_FLNMS:
                 assert assert_fname(p.save(fname)[0])
 
+tgal = 'TipsyGalaxy/galaxy.00300'
+@requires_file(tgal)
+def test_particle_phase_plot_semantics():
+    ds = load(tgal)
+    ad = ds.all_data()
+    dens_ex = ad.quantities.extrema(('Gas', 'density'))
+    temp_ex = ad.quantities.extrema(('Gas', 'temperature'))
+    plot = ParticlePlot(ds,
+                        ('Gas', 'density'),
+                        ('Gas', 'temperature'),
+                        ('Gas', 'particle_mass'))
+    plot.set_log('density', True)
```

Comment:
A small note is that trying to run this same line with the full field tuple causes a failure.  But this should not be considered a blocker for this PR--just to point out that field identification is sometimes ambiguous.